notebooks/hive2bq/HiveToBigquery_notebook.ipynb

```diff
@@ -888,7 +929,7 @@
     "        migrate_hive(execution_list)\n",
     "    except Exception:\n",
     "        AUDIT_DICT[\"Job_Status\"]=\"FAIL\"\n",
-    "        print(\"\\n\\nSome Error Occured while loading Table Set: \"+str(execution_list))\n",
+    "        raise Exception(\"\\n\\nSome Error Occured while loading Table Set: \"+str(execution_list))\n",
```

Comment:
I think you are right to raise an exception instead of just printing a message. But this still loses the original exception. Instead you might want to re-raise from the original exception. e.g.:

```
except Exception as exc:
    raise Exception(exception_message) from exc
```